pandas/plotting/_misc.py

```diff
@@ -602,12 +602,15 @@ def lag_plot(series, lag=1, ax=None, **kwds):
     return ax
 
 
-def autocorrelation_plot(series, ax=None, **kwds):
-    """Autocorrelation plot for time series.
+def autocorrelation_plot(series, n_lags=None, ax=None, **kwds):
+    """Autocorrelation plot for time series. Prints p value
```

Comment:
See the contributing guide on docstrings. First line should be a one-line summary.